docs/getting_started.md

```diff
@@ -194,6 +194,30 @@ capitalized.
 - `CreationDate` - the date the document was created (added automatically by PDFKit)
 - `ModDate` - the date the document was last modified
 
+## Destination
```

Comment:
Doesn’t seem like this should go on the getting started page. Maybe destinations should have their own page? You could give more examples of where you can use them too.